src/main/java/uk/gov/hmcts/reform/orgrolemapping/controller/RefreshController.java

```diff
@@ -26,19 +31,23 @@
 public class RefreshController {
 
     @Autowired
-    public RefreshController(RefreshOrchestrator refreshOrchestrator) {
+    public RefreshController(RefreshOrchestrator refreshOrchestrator,
+                             JudicialRefreshOrchestrator judicialRefreshOrchestrator) {
         this.refreshOrchestrator = refreshOrchestrator;
+        this.judicialRefreshOrchestrator = judicialRefreshOrchestrator;
     }
 
     RefreshOrchestrator refreshOrchestrator;
 
+    JudicialRefreshOrchestrator judicialRefreshOrchestrator;
+
     @PostMapping(
             path = "/am/role-mapping/refresh",
             produces = V1.MediaType.MAP_ASSIGNMENTS,
             consumes = {"application/json"}
     )
     @ResponseStatus(code = HttpStatus.ACCEPTED)
-    @ApiOperation("refreshes role assignments")
+    //@ApiOperation("refreshes role assignments")
```

Comment:
isn't it required anymore?

Reply:
It is commented because we are not showing this endpoint in swagger. this is existing endpoint to refresh through batch job.

Reply:
ok